Tabloid/client/src/components/CategoryEditForm.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import { FormGroup, Label, Button, Form, Input } from "reactstrap";
+import { editCategory, getCategory } from "../modules/categoryManager";
+import { useHistory, Link, useParams } from "react-router-dom";
+
+export const CategoryEditForm = () => {
+  const { categoryId } = useParams();
+  const [category, setCategory] = useState({
+    id: 1,
+    name: "",
+  });
+
+  const history = useHistory();
+
+  useEffect(() => {
+    if (categoryId) {
+      getCategory(categoryId)
+      .then((res) => { 
+          setCategory(res)
+      })}
+    }, [categoryId])
+
+
+    const edittingCategory = (evt) => {
+        evt.preventDefault();
+
+        const edittedCategory = {
```

Comment:
This object is not necessary since it's an exact copy of the category object in state.

Reply:
Took care of this on the comment below.